app/grants/templates/grants/hall_of_fame.html

```diff
@@ -0,0 +1,208 @@
+                            <div class="font-caption">{{ g.funded_by }}</div>
+                            <div class="font-title mt-3">{{ g.amount }}</div>
+                            <div class="font-caption text-grey-400">{% trans 'raised on Gitcoin' %}</div>
+                        </div>
+                        <div class="col-xs-12 col-sm-3 mt-4">
+                            <h5 class="font-body">{% trans 'Project Description' %}</h5>
+                            <div>{{ g.description }}</div>
+                        </div>
+                        <div class="col-xs-12 col-sm-5 mt-4">
+                            <h5 class="font-body">{% trans 'Accomplishment 1' %}</h5>
+                            <div class="font-body">{{ g.accomplishment_1 }}</div>
+                            <h5 class="font-body mt-4">{% trans 'Accomplishment 2' %}</h5>
+                            <div class="font-body">{{ g.accomplishment_2 }}</div>
+                        </div>
+                    </div>
+                    {% endfor %}
+                </div>
+            </div>
+
+            <div class="container-fluid">
+                <div class="d-block d-sm-none pt-4"></div>
+                <div class="d-none d-sm-block pt-5"></div>
+
+                <div class="container text-left">
+                    <div class="row">
+                        <div class="col-xs-12 col-sm-6 offset-sm-1 container pb-5">
+                            <div class="row">
+                                <h2 class="col">{%trans "Are you a Grantee?" %}</h2>
+                            </div>
+
+                            <div class="row">
+                                <div class="col mt-3 font-subheader">{%trans "We'd love to hear your journey. Share your story via email." %}
+                                </div>
+                            </div>
+
+                            <div class="d-block d-sm-none pt-3"></div>
+                            <div class="d-none d-sm-block pt-4"></div>
+
+                            <div class="row">
+                                <div class="col">
+                                    <a class="btn btn-primary" href="mailto:{{share_your_story_email}}">
+                                        Share Your Story
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-sm-5 d-none d-sm-block">
+                            <div class="float-right w-100 hof-letter">
+                                {% include 'grants/components/hall_of_fame_envelope.html' %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% include 'shared/footer_scripts.html' with slim=1 %}
+        {% include 'shared/footer.html' %}
+    </div>
+</body>
```

Comment:
Could we re-indent this with 1 tab == 2 spaces

Reply:
Also looks like we have a a few div which are empty.
Could we remove those ?

Reply:
This is done, empty divs are also removed.